src/test/java/ParserTest.java

```diff
@@ -0,0 +1,34 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import command.DeadlineCommand;
+import command.TodoCommand;
+import duke.Parser;
+import exception.DukeException;
+import exception.InvalidDateException;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.Test;
+
+public class ParserTest {
+
+  @Test
+  public void parseDateTest1() throws InvalidDateException {
```

Comment:
Perhaps the test method name could be better constructed to follow the format: featureUnderTest_testScenario_expectedBehavior()? I noticed the same issue in most of the other test methods as well.